src/containers/TheHeader.js

```diff
@@ -79,6 +79,11 @@ const TheHeader = () => {
               <i className="cib-discord"> </i> Discord Logged In
             </button>
           )}
+          <div style={{paddingLeft: 10}}>
```

Comment:
Could you check if the `paddingLeft` is optically aligned even when the page is checked for responsive sizes?

Reply:
yes it is

Reply:
these buttons are not visible on mobile